test/unit/logic/transactionPool.js

```diff
@@ -13,82 +13,600 @@
  */
 'use strict';
 
-var lisk = require('lisk-js');
+var _ = require('lodash');
```

Comment:
@Tschakki please do not require `_`  `chai` and `sinon` all of these are available in the global context.